src/app/static/src/app/components/downloadResults/UploadModal.vue

```diff
@@ -249,31 +293,47 @@
                     return [];
                 }
             },
+            uploadDisabled(){
+                if (this.choiceUpload === 'uploadFiles' && this.uploadFilesToAdr.length < 1){
+                    return true
+                } else {
+                    return false
+                }
+            },
             downloadingFiles() {
                 return !!this.spectrum.downloading || !!this.summary.downloading;
             }
         },
         components: {
             Modal,
+            HelpCircleIcon,
             DownloadProgress
         },
-      watch: {
-        uploadFiles() {
-          this.setDefaultCheckedItems();
+        watch: {
+            uploadFiles() {
```

Comment:
I tried putting `v-if="uploadModalOpen"` on the modal and then calling `this.setDefaultCheckedItems()` on mount and it seems to work properly when I manually tested it, but half of the unit tests will fail for strange reasons, such as not getting the correct translations. Do you have any idea why that might be?

Reply:
Which unit tests were failing, the ones for the UploadModal or for DownloadResults?

Reply:
I've got all tests passing now. They were originally set up in quite a hacky way because they couldn't rely on the checkboxes being set by the watch method so I think they make more sense now. The translations not coming through was very odd though. In one case, I needed to put an `await Vue.nextTick()` to get them to register on time for the test.